exasol_integration_test_docker_environment/test/test_test_environment_db_disk_size.py

```diff
@@ -46,21 +39,25 @@ def assert_disk_size(self, size: str):
 
     def test_default_db_disk_size(self):
         self.docker_environment_name = "test_default_db_disk_size"
-        self.docker_envs = self.test_environment.spawn_docker_test_environment(self.docker_environment_name)
+        docker_envs = self.test_environment.spawn_docker_test_environment(self.docker_environment_name)
         self.assert_disk_size("2 GiB")
+        close_environments(*docker_envs)
```

Comment:
This doesn't work, because if the assert hits, it throws a exception and skips the close. Furthermore, we currently have also no save guard, because we don't cleanup the Env on __del___.
Furthermore, maybe we want to introduce a context manager for the environment